draft-ietf-rats-eat.md

```diff
@@ -552,10 +552,11 @@ between manufacturers).
 
 There are privacy considerations for UEID's. See {{ueidprivacyconsiderations}}.
 
-The UEID should be permanent. It should never change for a given
-device / entity. In addition, it should not be reprogrammable.  UEID’s
-are variable length. All implementations MUST be able to receive
-UEID's that are 33 bytes long (1 type byte and 256 bits).  The
+The UEID is permanent. It never change for a given
```

Comment:
FDO IOT standard has reached a final state:  https://fidoalliance.org/specs/FDO/fido-device-onboard-v1.0-ps-20210323/fido-device-onboard-v1.0-ps-20210323.html#DevAttnProt.  Removing the 'should' requirement will mean it is not compliant with EAT.  Would recommend leaving text as is.

Reply:
A couple points here Giri,

EAT's definition of the UEID as permanent has stood for a long time and we shouldn't really change that underneath of other implementations. FIDO IoT is not the only user of EAT.

The solution of two identifiers, one permanent and one that changes on lifecycle events seems like a good one that can serve lots of use cases. It is validated by IDevID/LDevID doing the same. Removing the should would destroy this solution.

FIDO standards change all the time, so this could be changed. FIDO also knew the situation before finalizing. Is there a substantial installed base of concern here for which there is no workaround?

Is there anything wrong with SUEID in this PR that would make it unsuitable for FIDO IoT? Haven't heard any comment on that yet.

Reply:
No issue with SUEID.  Will take up the proposal with FIDO to get feedback.  

The other relevant text from Sec. 8.1:  

"The device authenticates the relying party and generates a derived UEID just for that particular relying party. "

This above is consistent with current FIDO usage, as the UEID is transitioned from manufacturer-assigned to the new relying party (in this case the Device Owner).  Or is SUEID meant to supplant the above?